code/graphics/opengl/gropengltexture.cpp

```diff
@@ -1701,7 +1701,8 @@ int opengl_set_render_target( int slot, int face, int is_static )
 		return 0;
 	}
 
-	if ( !glIsFramebuffer(fbo->framebuffer_id) /*|| !glIsRenderbufferEXT(fbo->renderbuffer_id)*/ ) {
+	// Since framebuffer_id is only ever 0 or assigned by glGenFramebuffer, it must be valid if not 0
+	if ( fbo->framebuffer_id == 0 /*!glIsFramebuffer(fbo->framebuffer_id)*/ /*|| !glIsRenderbufferEXT(fbo->renderbuffer_id)*/ ) {
```

Comment:
Removing the commented out code is probably a good idea, just to reduce some confusion for later readers